courseplay_event.lua

```diff
@@ -1,16 +1,21 @@
+local curFile = 'courseplay_event.lua';
 CourseplayEvent = {};
-CourseplayEvent_mt = Class(CourseplayEvent, Event);
+local CourseplayEvent_mt = Class(CourseplayEvent, Event);
 
 InitEventClass(CourseplayEvent, "CourseplayEvent");
 
 function CourseplayEvent:emptyNew()
+	print(string.format("CP EVENT: %s","receive new event"))
```

Comment:
Please always use courseplay:debug() or courseplay.debugVehicle() (if the vehicle is known) with channel 5 (multiplayer debug channel) instead of print.

Reply:
Hi @pvaiko thanks for this will do, was my own debugging I added and forgot to remove. 

I notice that the AI vehicle commands/events still needs to be implemented. Or rather the broadcasts on client side. I am willing to help with this. Courseplay being one of our favourite mods in fs17 and I only got fs19 like 2 weeks ago we surely miss it in MP. 

Let me know where I can help. I think its a good start with regards to that its not crashing MP. Only thing missing I picked up is the broadcast to server to start the AI system.